RELEASE.md

```diff
@@ -0,0 +1,34 @@
+Release type: patch
+
+Adds support for converting pydantic conlist.
+Note that constraint is not enforced in the graphql type.
+Thus, we recommend always working on the pydantic type such that the validated is enforced.
+
+```python
+import strawberry
+from pydantic import BaseModel, conlist
+
+class Example(BaseModel):
+    friends: conlist(str, min_items=1)
+
+@strawberry.experimental.pydantic.type(model=Example, all_fields=True, is_input=True)
```

Comment:
To be consistent with the other docs it might be good to change this to use `strawberry.experimental.pydantic.input` instead of passing in `is_input`